<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenSwitchWidget</class>
 <widget class="QWidget" name="ScreenSwitchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>19</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="left_toolButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>511</width>
        <height>138</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="right_toolButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>left_toolButton</sender>
   <signal>clicked()</signal>
   <receiver>ScreenSwitchWidget</receiver>
   <slot>slotForward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>15</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>-13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>right_toolButton</sender>
   <signal>clicked()</signal>
   <receiver>ScreenSwitchWidget</receiver>
   <slot>slotBackward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>585</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>-12</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotForward()</slot>
  <slot>slotBackward()</slot>
 </slots>
</ui>
